codec-http2/src/test/java/io/netty/handler/codec/http2/Http2ConnectionRoundtripTest.java

```diff
@@ -649,6 +653,79 @@ public void run() throws Http2Exception {
         assertFalse(clientChannel.isOpen());
     }
 
+    private enum WriteEmptyBufferMode {
+        SingleEndOfStream,
```

Comment:
this is rather surprising, I have almost never seen enums not use CONSTANT_CASE

Reply:
updated